docs/data-management/delete.md

```diff
@@ -95,9 +95,11 @@ The available grammar is:
     "id": <task_id>,
     "dataSource": <task_datasource>,
     "interval" : <all_unused_segments_in_this_interval_will_die!>,
-    "context": <task context>
+    "context": <task context>,
+    "batchSize": <optional batch size, default is 100. Too large will affect overlord stability.>
```

Comment:
can you add that the reason is due to the locking

Reply:
fixed in d3d5a71ddf41dd258a7721b8bd1893b25e83cd11